src/main/io/vtx_string.c

```diff
@@ -43,23 +45,20 @@ const char * const vtx58BandNames[] = {
     "BOSCAM E",
     "FATSHARK",
     "RACEBAND",
+    "LOW RACE",
 };
 
-const char vtx58BandLetter[] = "-ABEFR";
+const char vtx58BandLetter[] = "-ABEFRL";
 
 const char * const vtx58ChannelNames[] = {
     "-", "1", "2", "3", "4", "5", "6", "7", "8",
 };
 
 bool vtx58_Freq2Bandchan(uint16_t freq, uint8_t *pBand, uint8_t *pChannel)
 {
-    int8_t band;
-    uint8_t channel;
-
-    // Use reverse lookup order so that 5880Mhz
-    // get Raceband 7 instead of Fatshark 8.
-    for (band = 4 ; band >= 0 ; band--) {
-        for (channel = 0 ; channel < 8 ; channel++) {
+    // Use reverse lookup order so that 5880MHz get RaceBand 7 instead of FatShark 8.
+    for (int8_t band = ARRAYLEN(vtx58frequencyTable) - 1 ; band >= 0 ; band--) {
```

Comment:
(use `int`)
And maybe use VTX_SA_BAND_COUNT instead of ARRAYLEN

Reply:
I only joined variable declarations with assignments under `for` to prettify this code a bit without touching it's semantic, not sure I want to change the types in this perfectly working code with this pull request.